exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -134,20 +134,36 @@ export default ExpFrameBaseComponent.extend({
   }),
 
   // Represent the sorted cards in a human-readable format for storage in the database
-  responses: Ember.computed('cardSortResponse', function () {
-      let cardSortResponse = this.get('cardSortResponse');
-
-      // Final data should be returned as object {categoryName: [cardIdentifiers] }
-      let responses = {};
-
-      // Assumption: this unpacks a list of { categories: {name: name, cards: [cards]} } objects
-      for (let categorySet of cardSortResponse) {
-          for (let category of categorySet.categories) {
-              let name = category.name.split('.').pop();
-              responses[name] = category.cards.map((cardItem) => cardItem.id);
+  responses: Ember.computed('cardSortResponse', 'cardSortResponse2', {
+      get(key) {
+          // Final data for each card-sort should be returned as object {categoryName: [cardIdentifiers] }
+          let responses = {
+              cardSort1: {},
+              cardSort2: {}
+          };
+          let cardSortResponse;
+          if (this.get('cardSortResponse')) {
+              cardSortResponse = this.get('cardSortResponse');
+              for (let category of cardSortResponse) {
+                  let name = category.name.split('.').pop();
+                  responses['cardSort1'][name] = category.cards.map((cardItem) => cardItem.id);
+              }
+          }
+          if (this.get('cardSortResponse2')) {
+              cardSortResponse = this.get('cardSortResponse2');
+              // Assumption: this unpacks a list of { categories: {name: name, cards: [cards]} } objects
+              for (let categorySet of cardSortResponse) {
+                  for (let category of categorySet.categories) {
+                      let name = category.name.split('.').pop();
+                      responses['cardSort2'][name] = category.cards.map((cardItem) => cardItem.id);
+                  }
+              }
           }
+          return responses;
+      },
+      set(key, value) {
```

Comment:
Overall, if a user exits out of the survey before completing it and then they log back in, they should see all of their previous answers. 

So the setter (which should be a TODO because I'm still writing it!) will need to de-serialize the expData and update the values of the buckets to show the user's previous answers.

But now I'm thinking, maybe it would be easier to take the user directly to the page they were at before exiting?